### ISSUE TITLE
Node search box disappears when the cursor is not near the box

### ISSUE THREAD

**Issue by oxysoft** [comfyanonymous/ComfyUI#1274]
Double click on the canvas to appear the node searching box, place the mouse away and let go - it will disappear.

**Comment by txirrindulari** [comment]
It is very frustrating if you use a wacom, the pen pointer does not stay still like a mouse pointer

**Comment by halr9000** [comment]
Yeah it seems like this was added recently as I don't recall it doing that a month or so back. I tried to get used to it, but it's still tripping me up, so I am not a fan. Would rather it stay open, and if I want it gone, I can always hit esc or click elsewhere. Maybe make it optional.
